src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -156,21 +152,23 @@ private static void cloneAndAnalyzeRepos(List<RepoConfiguration> configs, String
             if (clonedRepoLocation == null) {
                 handleCloningFailed(configs, currRepoLocation);
             } else {
-                analyzeRepos(outputPath, configs, repoLocationMap.get(clonedRepoLocation),
-                        repoCloner.getCurrentRepoDefaultBranch());
+                generatedFiles.addAll(analyzeRepos(outputPath, configs, repoLocationMap.get(clonedRepoLocation),
+                    repoCloner.getCurrentRepoDefaultBranch()));
             }
             currRepoLocation = nextRepoLocation;
         }
         repoCloner.cleanup();
+        return generatedFiles;
     }
 
     /**
      * Analyzes all repos in {@code configsToAnalyze} and generates their report.
      * Also removes {@code configsToAnalyze} that failed to analyze from {@code configs}.
      */
-    private static void analyzeRepos(String outputPath, List<RepoConfiguration> configs,
+    private static List<Path> analyzeRepos(String outputPath, List<RepoConfiguration> configs,
```

Comment:
Same for this javadoc comment

Reply:
Done. I've indicated the return value in the Javadoc.